test/test_art.py

```diff
@@ -416,6 +401,112 @@ def test_google_art_returns_no_result_with_malformed_response(self):
+          "250": "http://coverartarchive.org/release/rid/12345-250.jpg",
+          "500": "http://coverartarchive.org/release/rid/12345-500.jpg",
+          "large": "http://coverartarchive.org/release/rid/12345-500.jpg",
+          "small": "http://coverartarchive.org/release/rid/12345-250.jpg"
+        },
+        "types": [
+          "Front"
+        ]
+      },
+      {
+        "approved": false,
+        "back": false,
+        "comment": "",
+        "edit": 12345,
+        "front": false,
+        "id": 12345,
+        "image": "http://coverartarchive.org/release/rid/12345.jpg",
+        "thumbnails": {
+          "1200": "http://coverartarchive.org/release/rid/12345-1200.jpg",
+          "250": "http://coverartarchive.org/release/rid/12345-250.jpg",
+          "500": "http://coverartarchive.org/release/rid/12345-500.jpg",
+          "large": "http://coverartarchive.org/release/rid/12345-500.jpg",
+          "small": "http://coverartarchive.org/release/rid/12345-250.jpg"
+        },
+        "types": [
+          "Front"
+        ]
+      }
+    ],
+    "release": "https://musicbrainz.org/release/releaseid"
+}"""
+    RESPONSE_GROUP = """{
+        "images": [
+          {
+            "approved": false,
+            "back": false,
+            "comment": "",
+            "edit": 12345,
+            "front": true,
+            "id": 12345,
+            "image": "http://coverartarchive.org/release/releaseid/12345.jpg",
+            "thumbnails": {
+              "1200": "http://coverartarchive.org/release/rgid/12345-1200.jpg",
+              "250": "http://coverartarchive.org/release/rgid/12345-250.jpg",
+              "500": "http://coverartarchive.org/release/rgid/12345-500.jpg",
+              "large": "http://coverartarchive.org/release/rgid/12345-500.jpg",
+              "small": "http://coverartarchive.org/release/rgid/12345-250.jpg"
+            },
+            "types": [
+              "Front"
+            ]
+          }
+        ],
+        "release": "https://musicbrainz.org/release/release-id"
+    }"""
+
+    def setUp(self):
+        super(CoverArtArchiveTest, self).setUp()
+        self.source = fetchart.CoverArtArchive(logger, self.plugin.config)
+        self.settings = Settings(maxwidth=0)
+
+        if util.SNI_SUPPORTED:
+            self.RELEASE_URL = "https://" + self.RELEASE_URL
+            self.GROUP_URL = "https://" + self.GROUP_URL
+        else:
+            self.RELEASE_URL = "http://" + self.RELEASE_URL
+            self.GROUP_URL = "http://" + self.GROUP_URL
+
+    @responses.activate
+    def run(self, *args, **kwargs):
+        super(CoverArtArchiveTest, self).run(*args, **kwargs)
+
+    def mock_response(self, url, json):
+        responses.add(responses.GET, url, body=json,
+                      content_type='application/json')
+
+    def test_caa_finds_image(self):
+        album = _common.Bag(mb_albumid=self.MBID, mb_releasegroupid=self.MBID)
```

Comment:
Maybe use two different IDs here? E.g. `MBID_RELEASE` and `MBID_GROUP` in analogy to `RESPONSE_RELEASE` and `RESPONSE_GROUP`.

Reply:
Done.